cbdr/cortex
# Kill Legacy!

## Purpose:
This beastly PR removes almost every scrap of Legacy from Cortex. What remains are some RESTful endpoints so that Beta Cortex data can still be consumed in the short term, as well as the old Localization system, which may come in handy one day. Why are we splitting from Legacy? We're going to overhaul Users and Tenancy. Legacy makes too many assumptions about how Users/Tenancy works, and would need to be overhauled alongside these changes. This is too great a burden, so Legacy is officially going away.

## JIRA:
N/A

## Steps to Take On Prod
We will need new production servers for Legacy, as well as new subdomains/ssl certs for legacy. We will want to migrate Legacy data over to these new servers, as well. We'll want to hold off on all of us this until we bake in basic user management into the Beta Cortex interface. 

Edit: actually, we may want to create new servers for Beta Cortex, as deleting migrations means we need to re-create the DBs from scratch. Legacy stacks will simply be renamed. All Dev, Staging and Feature stacks will also need to be re-created.

## Changes:
* Changes to setup
  * Oh, you know... "stuff".

* Architectural changes
  * N/A

* Migrations
  * Yes! In fact, we've /removed/ Legacy migrations. This is mainly as a benefit to our future Open Source users, who don't want these crufty migrations present when hunting down problems.

* Library changes
  * N/A

* Side effects
  * N/A

## Screenshots
* Before
N/A

* After
N/A

## QA Links:
http://web.cortex-6.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * N/A

* Steps to take
  * Ensure you can login & Beta Cortex features still work as expected

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
N/A

## Additional Information
N/A
